repose-aggregator/components/services/datastore-service/datastore-service-api/src/main/java/org/openrepose/core/services/datastore/DatastoreService.java

```diff
@@ -73,15 +78,33 @@ DistributedDatastore createDatastore(String datastoreName, ClusterConfiguration
      * Create and return a distributed datastore using the provided configuration.  The created
      * datastore can be retrieved by the same name provided using getDatastore(datastoreName)
      *
-     * @param datastoreName
-     * @param configuration
-     * @return
+     * @param datastoreName unique name for this Datastore
+     * @param configuration Configuration for the entire Cluster
+     * @param connPoolId    the name of the pool to borrow a connection from
+     * @param useHttps      indicates if SSL/TLS should be used
+     * @return the newly created Datastore
      * @throws DatastoreServiceException if the datastore creation fails
      */
     DistributedDatastore createDistributedDatastore(String datastoreName, ClusterConfiguration configuration, String connPoolId, boolean useHttps)
             throws DatastoreServiceException;
 
     /**
+     * Create and return a remote datastore using the provided configuration.  The created
+     * datastore can be retrieved by the same name provided using getDatastore(datastoreName)
+     *
+     * @param datastoreName    unique name for this Datastore
+     * @param proxyService     Proxy Service for making remote calls
+     * @param encodingProvider Encoding Provider
+     * @param target           the IP Socket Address of the Datastore
+     * @param connPoolId       the name of the pool to borrow a connection from
+     * @param useHttps         indicates if SSL/TLS should be used
+     * @return the newly created Datastore
```

Comment:
Right. All of the `@param`'s line up. If there were multiple `@return`'s, which would be way bad and wrong, they would line up. The one's with multiple `@throw`'s should line up too.